e2e/test/scenarios/dashboard-filters/dashboard-chained-filters.cy.spec.js

```diff
@@ -87,28 +87,30 @@ describe("scenarios > dashboard > chained filter", () => {
         }
       });
 
-      H.popover()
-        .last()
-        .within(() => {
-          cy.findByText("Anchorage");
-          cy.findByText("Anacoco").should("not.exist");
-        });
+      const valuesWidget = () =>
+        has_field_values === "search"
+          ? cy.findByRole("listbox")
+          : cy.findByTestId("field-values-widget");
```

Comment:
I had the same thought, but the effort/value ratio of introducing the right kind of abstraction doesn't seem worth it to me, at the moment. Wdyt?

Reply:
well yeah I respect that. I just wanna know the reason, feel free to keep it as is.

Reply:
I'm not sure if this is a good enough answer, but I went with this approach because there wasn't enough in common between the two values-containing widgets to use the same selector in both runs of this test.

The previous version of the test relied on the order of the popovers in the DOM to pick out the right one. That no longer works, because I've altered the order of popovers, and of course it's also not particularly semantic.